abb_driver/rapid/ROS_festo.sys

```diff
@@ -0,0 +1,33 @@
+MODULE Festo_Module(SYSMODULE)
+
+	PROC Festo_init()
+        TPWrite "Init Festo";
+		SETGO CPOS_Start,0;
+		SETGO CCON,0;
+		WaitTime .1;
+		SETGO CCON,67;
+		SETGO CDIR,0;
+		SETGO Velocity,0;
+		SETGO Position,0;
+		WaitTime .1;
+	ENDPROC
+
+    PROC Festo_Homing()
+        TPWrite "Home Festo";
+        SetGO CPOS_Start,5;
+    ENDPROC
+
+    PROC Festo_move()
+        SetGO CPOS_Start,1;
+        SETGO Velocity,FESTO_rail_velocity;
+        TPWrite "written festo pos: " \Num:=FESTO_rail_position;
+        IF(FESTO_rail_position > 2500) THEN FESTO_rail_position := 2499; ENDIF

```

Comment:
make IF clause equal to the below one? 
`IF(FESTO_rail_position > 2500) THEN FESTO_rail_position := 2500`
or
`IF(FESTO_rail_position > 2499) THEN FESTO_rail_position := 2499`

Reply:
Suggestion: refactor checking and enforcing joint limits into a separate function.